### PR TITLE
Support joint consensus

### PR DESCRIPTION
**Pull request by NingLin-P**

### CODE REVIEW

**Comment by NingLin-P** [comment]
PTAL @BusyJay @nolouch

File: proto/raft_cmdpb.proto [hunk 1]
```diff
@@ -215,6 +219,7 @@ message AdminRequest {
     CommitMergeRequest commit_merge = 8;
     RollbackMergeRequest rollback_merge = 9;
     BatchSplitRequest splits = 10;
+    ChangePeerV2Request change_peer_v2 = 11;
```
Comment (BusyJay) [inline proto/raft_cmdpb.proto:222]: I suggest to add a new `AdminCmdType` to avoid potential compatibility bug.

File: proto/raft_cmdpb.proto [hunk 2]
```diff
@@ -215,6 +220,7 @@ message AdminRequest {
...
```
Comment (BusyJay) [inline proto/raft_cmdpb.proto:223]: Should add corresponding response too.

File: proto/raft_cmdpb.proto [hunk 3]
```diff
@@ -116,6 +116,14 @@ message ChangePeerResponse {
     metapb.Region region = 1;
 }
 
+message ChangePeerV2Request {
+    repeated ChangePeerRequest changes = 1;
+}
+
+message ChangePeerV2Response {
```
Comment (BusyJay) [inline proto/raft_cmdpb.proto:123]: It's not used.

Comment (NingLin-P) [reply proto/raft_cmdpb.proto:123]: Fixed.

**Pull request merged**